### ISSUE TITLE
Empty collapsed_feature_tables

### ISSUE THREAD

**Issue by mishkb** [MaestSi/MetONTIIME#22]
Hi Simone,
I have been trying to use MetONTIIME to analyse fastq.gz that have already been demultiplexed and basecalled with Guppy. 
I have tried using both Vsearch and Blast, and various Query Cover and ID thresholds - most recently inlcuding the values of QC=0.3 and ID_thr=0.77 as recommended in the config_MinION_mobile_lab.R file. However I have never been able to get any data in the files in the collapsed_feature_tables - which I think I need in order to generate taxa plots.

I have attached the nohub.out file - could you please help me interpret the error(s) I'm getting?

The MetONTIIME script generates a sequences.qza artifact, manifest.txt file, demux_summary.qzv artifact and a folder called callapsed_feature_tables which contains 7 files (feature-table_absfreq_level1_stringent.tsv etc) that are all empty.

The data that is generated indicates that the MetONTIIME script is correctly reading the fastq files because the #of reads per barcoded sample are correct (from demux_summary.qzv). 

Thanks for any guidance you can offer!
[nohup.txt](https://github.com/MaestSi/MetONTIIME/files/5676413/nohup.txt)

**Comment by MaestSi** [comment]
Hi, in the nohup file I found:

Invalid value for '--p-threads': received <Blast> as an argument, which
  is incompatible with parameter type: Int % Range(0, 256, inclusive_end=True)

Are you specifying the number of threads parameter, in the command line? Moreover, are you specifying the path to the files in the command line or just the file name?
Simone

**Comment by mishkb** [comment]
Thanks for your suggestions Simone. I  did specify threads and did have the full path for the files in the command - but I was running into problems because I had mistakenly left off the path for the metadata file in the command, so the thread parameter was parsed as the classifier name. 
The script is still running now (I'm trying Blast first, which is only single thread) - but already is further along than previously.
I also adjusted the ID_THR as you suggested - I'll come back to this and test on some different values if I get the time. Currently running some sequences that came from a Flongle (which I have found to have a lower mean Q score overall). 
I really appreciate how quickly you have responded to my questions.
Michelle
